client-libraries/rust/examples/utils_mutator.rs

```diff
@@ -0,0 +1,97 @@
+use std::{
+    collections::{BTreeMap, HashMap},
+    sync::{atomic::AtomicI64, Arc},
+    time::Duration,
+};
+
+use auxon_sdk::{
+    api::AttrVal,
+    mutation_plane::types::{MutationId, MutatorId},
+    mutator_protocol::descriptor::owned::{OwnedMutatorDescriptor, OwnedMutatorParamDescriptor},
+    plugin_utils::{config::Config, mutation::Mutator},
+};
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error + Sync + Send>> {
+    auxon_sdk::init_tracing!();
+
+    let value = Arc::new(AtomicI64::new(0));
+
+    println!("Connecting...");
+    let config = Config::load_common()?;
+    let mut mh = config.connect_and_authenticate_mutation().await?;
+    mh.register_mutator(Box::new(OffsetNumberMutator::new(value.clone())))
+        .await?;
+    // TODO allow mutators to be added after connect
```

Comment:
I don't see this scenario being explicitly rejected in the impl, is it actually unsupported?

Reply:
It's a stale comment, I actually fixed this.